scripts/scil_detect_streamlines_loops.py

```diff
@@ -76,36 +82,36 @@ def main():
         parser.error('Angle "{}" '.format(args.angle) +
                      'must be greater than 0')
 
-    tractogram = nib.streamlines.load(args.in_tractogram)
+    tractogram = load_tractogram_with_reference(
+        parser, args, args.in_tractogram)
+
     streamlines = tractogram.streamlines
 
-    streamlines_c = []
-    loops = []
+    ids_c = []
+
+    ids_l = []
+
     if len(streamlines) > 1:
-        streamlines_c, loops = remove_loops_and_sharp_turns(streamlines,
-                                                            args.angle,
-                                                            args.qb,
-                                                            args.threshold)
+        ids_c = remove_loops_and_sharp_turns(
+            streamlines, args.angle, args.qb, args.threshold)
```

Comment:
Please use `use_qb=args.qb` for optional arguments... same for decompose_connectivity